debian/control

```diff
@@ -2,7 +2,7 @@ Source: phd2
 Section: education
 Priority: optional
 Maintainer: Patrick Chevalley <pch@ap-i.net>
-Build-Depends: debhelper (>= 9), cmake, libwxgtk3.2-dev | libwxgtk3.0-dev | libwxgtk3.0-gtk3-dev, libcfitsio-dev, libopencv-dev, libusb-1.0-0-dev, libudev-dev, libv4l-dev, libnova-dev, libcurl4-gnutls-dev, libindi-dev (>= 2.0), libeigen3-dev, libgtest-dev libopencv-dev
+Build-Depends: debhelper (>= 9), cmake, libwxgtk3.2-dev | libwxgtk3.0-dev | libwxgtk3.0-gtk3-dev, libcfitsio-dev, libopencv-dev, libusb-1.0-0-dev, libudev-dev, libv4l-dev, libnova-dev, libcurl4-gnutls-dev, libindi-dev (>= 2.0), libeigen3-dev, libgtest-dev
```

Comment:
I assume you did not mean to remove libopencv-dev

Reply:
Yes  this is intended because when I add it I not remark it was already there for a long time, just after libcfitsio-dev.

Reply:
oh intended after all. ok